administrator/components/com_menus/tmpl/item/edit_container.php

```
@@ -41,67 +41,66 @@
 		</div>
 
 		<hr>
-
-		<ul class="treeselect">
+		<div class="alert alert-info">
+			<span class="fa fa-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
+			<?php echo Text::_('COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC'); ?>
+		</div>
 			<?php if (count($menuLinks)) : ?>
```

If before the alert div

---

No that is not correct

COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Select the menu items that should or should not be shown under this container. If there are no items to show, then this container will also be hidden.<br>Please note that when you install a new component it will be displayed by default until you come back here and hide it too."

---

Ahh so the existing behaviour of showing it was wrong. Fair enough